### PR TITLE
naprawione readme

### PR DESCRIPTION
**Pull request by Karolingo**

### CODE REVIEW

File: discord_bot/README.md [hunk 1]
```diff
@@ -3,23 +3,24 @@
 ## Setup
 
 ```bash
-# Create virtual env
-python3 -m venv .venv
-source .venv/bin/activate
+### Clone repository
+direct link - **https://github.com/knmlprz/ChatKNML.git**
```
Comment (TheJimmyNowak) [inline discord_bot/README.md:7]: Użyj git clone

File: discord_bot/README.md [hunk 2]
```diff
@@ -3,23 +3,24 @@
...
 
-# Install packages
-pip install -r requirements.txt
+### Install docker compose
+direct link - **https://docs.docker.com/desktop/install/windows-install/**
 
-# Create env variables
-touch .env
+### After docker has been installed, make sure to register and log in to docker desktop.
```
Comment (TheJimmyNowak) [inline discord_bot/README.md:12]: Use warning quote from github markdown docs

File: discord_bot/README.md [hunk 3]
```diff
@@ -3,23 +3,24 @@
 ## Setup
 
 ```bash
```
Comment (TheJimmyNowak) [inline discord_bot/README.md:5]: Delete this

File: discord_bot/README.md [hunk 4]
```diff
@@ -3,23 +3,24 @@
 ## Setup
```
Comment (TheJimmyNowak) [inline discord_bot/README.md:3]: Use headear 1

File: discord_bot/README.md [hunk 5]
```diff
@@ -3,23 +3,24 @@
...
+direct link - **https://docs.docker.com/desktop/install/windows-install/**
 
-# Create env variables
-touch .env
+### After docker has been installed, make sure to register and log in to docker desktop.
 
-# Insert Bot token here
-echo "[BOT TOKEN]" > .env
+### Install poetry and packs needed
+pip poetry install
```
Comment (TheJimmyNowak) [inline discord_bot/README.md:15]: Do not install poetry using pip. 
```sh
pip install pipx
pipx install poetry
```

File: docker-compose.yml [hunk 6]
```diff
@@ -3,21 +3,7 @@ services:
...
-      - 5001:80
-    deploy:
-      resources:
-        reservations:
-          devices:
-            - driver: nvidia
-              count: 1
-              capabilities: [gpu]
-
+
```
Comment (TheJimmyNowak) [inline docker-compose.yml:6]: Use `git reset --soft ~` and fix this

File: discord_bot/README.md [hunk 7]
```diff
@@ -3,23 +3,24 @@
...
+### Copy insides of ".env.example", create ".env" file, paste it there and put your own discord token in parenthesis
+TOKEN="DISCORD_TOKEN"
 
-# Run a bot!
-python3 Bot.py
+### Setting docker compose
+docker compose up
+
+### Run a bot!
+from "/discord_bot/discord_bot" path run bot.py
```
Comment (TheJimmyNowak) [inline discord_bot/README.md:25]: Improve this pls

**Review by TheJimmyNowak — CHANGES_REQUESTED** [review]

**Review by TheJimmyNowak — APPROVED** [review]
lgtm

**Review by finloop — APPROVED** [review]
LGTM

**Pull request merged**